util/keyfile/src/lib.rs

```diff
@@ -59,7 +61,50 @@ pub fn read_pubfile_data<R: std::io::Read>(
     Ok(result)
 }
 
-fn to_io_error(err: serde_json::error::Error) -> std::io::Error {
+/// Write user b58 public address to disk
+pub fn write_b58pubfile<P: AsRef<Path>>(
+    path: P,
+    addr: &PublicAddress,
+) -> Result<(), std::io::Error> {
+    let mut wrapper = PrintableWrapper::new();
+    wrapper.set_public_address(addr.into());
+
+    let data = wrapper.write_to_bytes().map_err(to_io_error)?;
+    let data = bs58::encode(data).into_vec();
```

Comment:
Should use https://github.com/mobilecoinfoundation/mobilecoin/blob/master/api/src/display.rs#L43

Reply:
thanks